docs/bridge.md

```diff
@@ -0,0 +1,41 @@
+# 💸 Asset Bridge
+
+`jstz` maintains a persistent ledger of all accounts and their balances of L2 tez (stored as mutez).
+
+The `jstz` _bridge_ implements a bridge protocol that allows to transfer fungible tokens (namely [CTEZ](https://ctez.app/)) from Tezos to the `jstz` rollup (and back *soon*™️).
+
+::: danger
+⚠️ Withdrawals from `jstz` to Tezos is not supported ⚠️
+:::
+
+## Quick Start
```

Comment:
Overall this seems to be a bit redundant as a very similar description is already in the CLI.